custom/icds_reports/tasks.py

```diff
@@ -1143,3 +1178,25 @@ def _bust_awc_cache():
     for key in reach_keys:
         cache.delete(key)
     create_datadog_event('redis: delete dashboard keys', 'finish')
+
+
+# TODO create new queue
+@task
+def run_citus_experiment(fn, args, kwargs):
+    experiment_context = {
+        "function_name": fn.__name__,
+        'args': args,
+        'kwargs': kwargs,
+    }
+    experiment = DashboardQueryExperiment(name="Dashboard Query Experiment", context=experiment_context)
+    with experiment.control() as c:
+        c.record(fn(*args, **kwargs))
+
+    with experiment.candidate() as c:
```

Comment:
nit: reusing variable name, single letter variable name 👎 @stickler-ci, where are you?